VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/non-host-waiting-room/non-host-waiting-room.component.html

```diff
@@ -0,0 +1,207 @@
+        >
+          {{ 'participant-waiting-room.private-consultation.start' | translate }}
+        </button>
+
+        <button
+          [disabled]="!connected"
+          class="govuk-button govuk-button-blue"
+          id="openJoinPCButton"
+          data-module="govuk-button"
+          (click)="openJoinConsultationModal()"
+        >
+          {{ 'participant-waiting-room.private-consultation.join' | translate }}
+        </button>
+      </div>
+      <app-wait-for-connection-message *ngIf="!connected"></app-wait-for-connection-message>
+
+      <div *ngIf="canStartJoinConsultation">
+        <app-private-consultation-description
+          [isExpanded]="privateConsultationAccordianExpanded"
+          (accordianToggled)="toggleAccordian()"
+        ></app-private-consultation-description>
+      </div>
+
+      <div class="govuk-!-margin-top-9">
+        <app-support-contact-details [vhTeamPhoneNumber$]="phoneNumber$"></app-support-contact-details>
+      </div>
+    </ng-container>
+    <ng-container *ngIf="!isParticipant">
+      <div>
+        <button
+          class="govuk-button govuk-button--bigblue"
+          type="button"
+          id="joinPCButton"
+          data-module="govuk-button"
+          (click)="joinJudicialConsultation()"
+          [disabled]="!connected"
+        >
+          {{ 'joh-waiting-room.enter-consultation-room' | translate }}
+          <span *ngIf="numberOfJudgeOrJOHsInConsultation" id="numberOfJohsInConsultationBadge" class="badge">
+            {{ numberOfJudgeOrJOHsInConsultation }}
+          </span>
+        </button>
+      </div>
+      <app-wait-for-connection-message *ngIf="!connected"></app-wait-for-connection-message>
+      <div class="govuk-!-margin-top-9">
+        <app-support-contact-details [vhTeamPhoneNumber$]="phoneNumber$"></app-support-contact-details>
+      </div>
+    </ng-container>
+  </div>
+  <div class="govuk-grid-column-one-third" [ngClass]="{ 'no-mass': !showExtraContent }">
+    <div *ngIf="isParticipant">
+      <app-individual-participant-status-list
+        [attr.aria-label]="'participant-waiting-room.individual-participant-status-list-label' | translate"
+        [conference]="vhConference"
+      >
+      </app-individual-participant-status-list>
+    </div>
+    <div *ngIf="!isParticipant">
+      <app-judge-participant-status-list [conference]="vhConference"> </app-judge-participant-status-list>
+    </div>
+  </div>
+</div>
+
+<app-participant-chat *ngIf="hearing && showExtraContent && instantMessagingEnabled" [hearing]="hearing"></app-participant-chat>
+
+<app-select-media-devices
+  *ngIf="displayDeviceChangeModal"
+  [showAudioOnlySetting]="allowAudioOnlyToggle"
+  (shouldClose)="onSelectMediaDeviceShouldClose()"
+  #selectMediaDevices
+></app-select-media-devices>
+
+<ng-container *ngIf="isParticipant">
+  <app-start-private-consultation
+    *ngIf="displayStartPrivateConsultationModal"
+    [loggedInUser]="loggedInUser"
```

Comment:
The input is of type `VHParticipant` but loggedInUser is `LoggedParticipantResponse`
Use the `getLoggedInParticipant` selector